esper/assets/js/views/video.jsx

```diff
@@ -20,8 +20,9 @@ class VideoView extends React.Component {
       curTrack: null,
       lastTrackBox: -1,
       segStart: -1,
-	  activePage: 2 
+      activePage: 0 
     };
+   this._handlePaginationSelect = this._handlePaginationSelect.bind(this)
```

Comment:
If you instead define the function as
```
_handlePaginationSelect = (selectedPage) => {
   ...
}
```
Then you don't need to do the bind. Magic!